tests/openwisp2/settings.py

```diff
@@ -200,73 +199,47 @@
         'description': 'Bugged chart for testing purposes.',
         'unit': 'bugs',
         'order': 999,
-        'query': {'influxdb': "BAD"},
+        'query': query[TIMESERIES_DB['DATABASE']]['bad_test'],
     },
     'default': {
         'type': 'line',
         'title': 'Default query for testing purposes',
         'description': 'Default query for testing purposes',
         'unit': 'n.',
         'order': 999,
-        'query': {
-            'influxdb': (
-                "SELECT {field_name} FROM {key} WHERE time >= '{time}' AND "
-                "content_type = '{content_type}' AND object_id = '{object_id}'"
-            )
-        },
+        'query': query[TIMESERIES_DB['DATABASE']]['default'],
```

Comment:
What's the advantage of storing the queries for the default charts in a separate file?
I think these changes are not useful, if I'm missing something let me know.